scripts/generate_secrets.py

```diff
@@ -0,0 +1,48 @@
+import yaml
+import sys
+
+
+def get_file_contents(filename, trim=False):
+    with open(filename, 'r') as f:
+        data = f.read()
+        if trim:
+            data = data.rstrip('\r\n')
+    return data
+
+
+keys_folder = sys.argv[1]
+
+keys = {}
+
+keys['EQ_SUBMISSION_SDX_PUBLIC_KEY'] = get_file_contents(keys_folder + 'sdc-submission-encryption-sdx-public-key.pem')
```

Comment:
Path concatenation should be done with `os.path.join`, this will provide more flexibility with what input will be allowed (currently the trailing slash is required). Also, it's probably worth doing the concatenation in `get_file_contents` as currently every method is passing keys_folder.